src/tests/functional/shared_test_classes/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ ov_add_target(
         INCLUDES
             PUBLIC
                 "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+            PRIVATE
+                "${OpenVINO_SOURCE_DIR}/src/plugins/template/include"
```

Comment:
Why???

Reply:
Because we don't have access from test lib to template plugin include dir